spec/features/idv/steps/in_person_opt_in_ipp_spec.rb

```diff
@@ -416,6 +420,8 @@
     before do
       allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { false }
       allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+      allow_any_instance_of(ServiceProvider).to receive(:in_person_proofing_enabled).
+        and_return(true)
```

Comment:
I think this is going to be the most expedient way for now, and then later we can overhaul IdvHelper (see below).